sda-commons-server-kafka/build.gradle

```diff
@@ -10,21 +10,8 @@ dependencies {
 
   compile ('com.github.ftrossbach:club-topicana-core') {
     exclude group: 'org.slf4j', module: 'slf4j-log4j12'
-    exclude group: 'org.apache.zookeeper', module: 'zookeeper'
     exclude group: 'org.apache.kafka', module: 'kafka_2.12'
     exclude group: 'org.apache.kafka', module: 'kafka_2.11'
```

Comment:
It should be enough to only exclude kafka_2.12 and kafka-clients. There is no dependency to the other noted dependencies (see https://search.maven.org/artifact/com.github.ftrossbach/club-topicana-core/0.1.0/jar).

Reply:
Done as suggested. 👍